pkg/downloader/manager.go

```diff
@@ -323,6 +325,14 @@ func (m *Manager) downloadAll(deps []*chart.Dependency) error {
 			continue
 		}
 
+		if m.SkipDownloadIfExists {
+			name := filepath.Base(churl)
+			if _, err = os.Stat(filepath.Join(destPath, name)); err == nil {
+				fmt.Fprintf(m.Out, "Already exists locally %s from repo %s\n", dep.Name, dep.Repository)
```

Comment:
Part of the download process is to download to a `tmpcharts/` directory, and then delete the old charts and copy `tmpcharts` to `charts/`. Unfortunately that means the files that exist are deleted, but `tmpcharts/` is empty.

Reply:
could you please clarify? it checks if the chart exists in the `charts` folder, if it doesn't, then it downloads the chart to `tmpchart` directory and after move it to the `charts` folder